cellprofiler/utilities/version.py

```diff
@@ -92,10 +92,23 @@ def get_version():
         import cellprofiler.frozen_version
         return cellprofiler.frozen_version.version_string
 
+def get_dotted_version():
+    if not hasattr(sys, 'frozen'):
+        try:
+            return subprocess.check_output(["git", "describe", "--tags"])\
```

Comment:
It was a bit of a stopgap so that the title bar would say 2.2.0. I bet we can come up with some clever way of putting the version in one place. IMHO:

     python setup.py build_version

should do that and this code should go away.